tests/conftest.py

```diff
@@ -1,16 +1,26 @@
+import os
+import sys
 from typing import Optional
 
 import pytest
 
 from rabbitmq_amqp_python_client import (
     AddressHelper,
     AMQPMessagingHandler,
-    ClientCert,
     Environment,
     Event,
-    SslConfigurationContext,
+    PKCS12Store,
+    PosixClientCert,
+    PosixSslConfigurationContext,
+    WinClientCert,
+    WinSslConfigurationContext,
     symbol,
 )
+from rabbitmq_amqp_python_client.ssl_configuration import (
+    FriendlyName,
+)
+
+os.chdir(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Pycharm likes to set the working directory to `tests`, this sets it to the project root so it can find `.ci/certs`.